### ISSUE TITLE
How do I customize the uploader when using the default setup?

### ISSUE THREAD

**Issue by adityamenon** [Pasvaz/laravel-juploader#5]
Hello again =)

The default setup guide works beautifully. I would like to make a change where I specify the `->with_option('accept_file_types')`... I am lost as to where this should be done?

The view statically calls `Uploader\ButtonBar` and such classes... I tried putting in:

```
        $uploader = IoC::resolve('Uploader');
        $uploader->with_option('accept_file_types', "/.+$/i")->start();
```

in the controller that renders this particular view, but maybe the uploader in the view is still using a different instance.

TL:DR;
I'd like to specify some `with_option`s to the uploader, how do I do this with the default setup guide?

Thank you very much for the great bundle and the help!

**Comment by Pasvaz** [comment]
Actually you did it just fine.
Keep in mind that in this way you setup the options for the UploadServer (php), if you want to change the options of the jquery uploader (javascript client side) you should use this syntax:

``` php
Uploader\Javascripter::with_option('url','/upload/dbupload')
                          ->with_option('XXXX','YYYY')
                          ->activate_uploader()
```

**Comment by Pasvaz** [comment]
Btw sorry for the delay, I don't know why I didn't get/notice this notification before.
